### PR TITLE
More minor isolated fixes

### PR DESCRIPTION
**Pull request by Roman971**
Another batch of minor fixes (changes described with each commit's message)

### CODE REVIEW

**Review by engineer124 — APPROVED** [review]
`Staves` feels weird to me instead of `Staffs`, but it seems correct

File: include/variables.h [hunk 1]
```diff
@@ -202,8 +202,6 @@ extern volatile OSTime gRDPTotalTime;
 
 extern ActiveSound gActiveSounds[7][MAX_CHANNELS_PER_BANK]; // total size = 0xA8
 extern u8 gSoundBankMuted[];
-extern u8 D_801333F0;
-extern u8 gAudioSfxSwapOff;
```
Comment (engineer124) [inline include/variables.h:206]: Could you also remove these externs in `code_800EC960.c`? There seems to be a third set of unneeded externs for these variables

File: src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.c [hunk 2]
```diff
@@ -55,7 +55,7 @@ static InitChainEntry sInitChain[] = {
 };
 
 void func_808C1190(s16* arg0, u8* arg1, s16 arg2) {
-    if (arg2[arg1] != 0) {
+    if (arg1[arg2] != 0) {
```
Comment (Dragorn421) [inline src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.c:58]: what the heck
I don't really have a comment or question but how did `arg2[arg1]` work or match

**Review by Dragorn421 — COMMENTED** [review]
Could we consider addressing the grammatically incorrect plural "Staffs" by changing `AudioOcarina_ResetStaffs` to `AudioOcarina_ResetStaffStructs` instead? or `AudioOcarina_ResetStaffData`? or something else

File: src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.c (hunk 2 above)
Comment (Roman971) [reply src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.c:58]: For some reason `index[array]` is completely allowed and defined in the C standard... And it doesn't usually affect codegen afaik (I don't think we needed it anywhere at least)

**Comment by Roman971** [comment]
I reverted the musical "Staffs" -> "Staves" change since both plural seem correct according to many sources

**Review by engineer124 — APPROVED** [review]

**Review by Dragorn421 — APPROVED** [review]

**Review by fig02 — APPROVED** [review]

**Pull request merged**